content/en/releases/patch-releases.md

```diff
@@ -1,15 +1,7 @@
 ---
 title: Patch Releases
 type: docs
-auto_generated: true
 ---
-<!-- THIS CONTENT IS AUTO-GENERATED via ./scripts/update-release-info.sh in k/website -->
-
-{{< warning >}}
-This content is auto-generated and links may not function. The source of the document is located [here](https://github.com/kubernetes/sig-release/blob/master/releases/patch-releases.md).
-{{< /warning >}}
-
-# Kubernetes Patch Releases
```

Comment:
I assume we deprecate https://github.com/kubernetes/sig-release/blob/master/releases/patch-releases.md and link it into this location, right?

Reply:
If we edit https://github.com/kubernetes/sig-release/blob/master/releases/patch-releases.md to mention deprecation or a link to the canonical page, we'll need to automate _removing_ that deprecation text / link from the copy on https://kubernetes.io/

Reply:
(I'm fine with doing that, but it'd need to happen)

Reply:
@sftim @saschagrunert -- That's the plan! (mentioned in the PR description)

>     * releases: Canonicalize patch release information on k/website
>     * releases: Canonicalize Release Managers page on k/website

> /hold for:
> 
>     * [ ]  discussion
>     * [ ]  approval of future patch schedule
>     * [ ]  accompanying tombstones in k/sig-release

The script has already been updated in support of that (ref: https://github.com/kubernetes/website/pull/27997/commits/39796d148141728307f3bcdd5f7e149a596b80a3).
Tombstones to follow.